src/integrationTest/java/uk/gov/hmcts/darts/task/service/AutomatedTaskServiceTest.java

```diff
@@ -593,9 +595,7 @@ void givenConfiguredTasksUpdateCronAndResetCronForProcessArmResponseFilesAutomat
 
         Optional<AutomatedTaskEntity> originalAutomatedTaskEntity =
             automatedTaskService.getAutomatedTaskEntityByTaskName(automatedTask.getTaskName());
-        log.info("TEST - Original task {} cron expression {}", automatedTask.getTaskName(),
-                 originalAutomatedTaskEntity.get().getCronExpression()
-        );
+        log.info("TEST - Original task {} cron expression {}", automatedTask.getTaskName(), originalAutomatedTaskEntity.get().getCronExpression());
 
         automatedTaskService.updateAutomatedTaskCronExpression(automatedTask.getTaskName(), "*/9 * * * * *");
```

Comment:
Is this Cron expression correct for "Daily @ 9PM".  Looking at https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/support/CronExpression.html
I expected the third asterisk to have `21`?